test/libsolidity/SolidityNatspecJSON.cpp

```diff
@@ -204,6 +204,75 @@ BOOST_AUTO_TEST_CASE(dev_and_user_no_doc)
 	checkNatspec(sourceCode, "test", userNatspec, true);
 }
 
+BOOST_AUTO_TEST_CASE(public_state_variable)
+{
+	char const* sourceCode = R"(
+		contract test {
+			/// @notice example of notice
+			/// @dev example of dev
```

Comment:
Can you check if `@returns` works here?

Reply:
Right now only `@dev` and `@notice` is allowed, but it could also make sense to allow `@return` here too.

Reply:
For contract
```solidity
contract test {
  /// @notice example of notice
  /// @dev example of dev
  /// @return returns something
  uint public state;
}
```
`devdoc` will now look like this:
```json
{
  "methods": {},
  "stateVariables":
  {
    "state":
    {
      "details": "example of dev",
      "return": "returns something"
    }
  }
}
```